### ISSUE TITLE
Dynamic drawing error for horizontal scroll

### ISSUE THREAD

**Issue by nozich** [Grsmto/simplebar#377]
Thx for simplebar, its awesome.

But, it isn't re-generating horizontal scroll for dynamically updated childs.

It working vertical scroll.
And it working horizontal scroll on re-appeared containers. (jQuery hide() and after show())

https://jsfiddle.net/wpkuht18/

**Comment by Grsmto** [comment]
Thank you very much for the reproducible example, that will help a lot to fix this! I'll look into it.

**Comment by Grsmto** [comment]
This will be fixed in the new major version. As this required a big change I preferred to defer it to a major release (simplebar@5.0.0).

**Comment by nozich** [comment]
Thank you so much, now its working with created elements,

But, :(

Its not working with (style changed) appeared elements.

:D https://jsfiddle.net/zfw0odxs/

**Comment by Grsmto** [comment]
Yes I'm aware of that, it's a limitation, we can't detect style changes and horizontal overflow does not trigger a resize, so there is no way for SimpleBar to recalculate when this happen. :(
The only way would be to have an interval checking for size but we want to avoid that.
That's one of the few cases where you will want to call the `recalculate` function manually.
